chart/values.yaml

```
@@ -109,7 +109,9 @@ images:
     pullPolicy: IfNotPresent
   redis:
     repository: redis
-    tag: 7-bookworm
+    # Redis is limited to 7-2-bookworm due to licencing change
+    # https://redis.io/blog/redis-adopts-dual-source-available-licensing/
+    tag: 7-2-bookworm
```

```suggestion
    tag: 7.2-bookworm
```

I think the tag has a period for 7.2 and not a hyphen 7-2
https://hub.docker.com/layers/library/redis/7.2-bookworm/images/sha256-370175fd0c43cc67ac208f08062e822e15b789e9eb75635ca7a54e31624a9ee2?context=explore

---

Ups

---

Luckily we have tests to catch those typos too :)